vendor/github.com/infobloxopen/infoblox-go-client/connector.go

```diff
@@ -331,6 +331,10 @@ func NewConnector(hostConfig HostConfig, transportConfig TransportConfig,
 	connector.Requestor.Init(connector.TransportConfig)
 
 	res = connector
```

Comment:
Dont make changes directly in the vendored code. Checkout infoblox-go-client fork, make changes and then when they gets merged, update the dependencies using glide.